packages/ui-drilldown/src/Drilldown/index.tsx

```diff
@@ -1126,7 +1126,7 @@ class Drilldown extends Component<DrilldownProps, DrilldownState> {
     if (subPageId) {
       optionProps.renderAfterLabel = <IconArrowOpenEndSolid />
       optionProps['aria-haspopup'] = true
-      optionProps.role = customRole || 'button'
+      optionProps.role = customRole || 'menuitem'
```

Comment:
Drilldown has `role=menu`, which should not have `role=button` children. We had a similar issue in the past: https://github.com/instructure/instructure-ui/pull/1831 Also see the W3 spec: https://w3c.github.io/aria/#menu